recipes/apischema/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "apischema" %}
+{% set version = "0.15.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wyfo/apischema/archive/v{{ version }}.tar.gz
+  sha256: 4313e92bdb833fd4bb650c0a0ca62f5d12542def8aa20ec041b60fcd9c59eb04
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    # dataclasses is only required for Python 3.6.  conda should be smart enough to not
+    # install it for Python >=3.7 in this noarch build - I hope.
```

Comment:
It is: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=dataclasses#empty-python-packages